addons/account/views/audit_trail_report_views.xml

```diff
@@ -27,7 +27,21 @@
                 <field name="account_audit_log_move_id"/>
                 <field name="author_id"/>
                 <field name="date" string="Date"/>
-                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
+                <filter string="Updated Entries" name="audit_updated_entries" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
+                <filter string="Cancelled"
+                        name="audit_cancelled"
+                        domain="[('tracking_value_ids.field_id.name', '=', 'state'),
+                                 ('tracking_value_ids.new_value_char', '=', 'Cancelled')]"
+                        groups="base.group_system"/>
+                <filter string="Posted"
+                        name="audit_posted"
+                        domain="[('tracking_value_ids.field_id.name', '=', 'state'),
+                                 ('tracking_value_ids.new_value_char', '=', 'Posted')]"
```

Comment:
Note that char values stored for selection fields use the key, not the value, so should probably be "posted". Using the value would be broken due to translations.

Reply:
No, It's using the value.
https://github.com/odoo/odoo/blob/cb31d4352c741a5f8a927a65debb8e027e87df22/addons/mail/models/mail_tracking_value.py#L95

example from testing:
```py
In [13]: ms[0].tracking_value_ids.new_value_char
Out[13]: 'Cancelled'
```

Reply:
Then it is basically broken due to translations. We should probalby store the key, not the value...

Reply:
How would you do that ? We may store "Posted" in any language, so don't see how we can do a filter on it, except maybe through a searchable field who would make something like "value, in, [all translated values]".

Reply:
For now we'll just skip creating the Posted and Cancelled filter then. I removed them